Scripts/autogenerated_files.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/sh
 
-GENERATED_DIR="./Wake\ On\ Lan/Modules/WOLResources/Sources/Generated"
+GENERATED_DIR="./Wake\ On\ Lan/Modules/WOLResources/Sources/WOLResources/Generated"
 eval mkdir -p $GENERATED_DIR
-eval touch $GENERATED_DIR/Strings+Generated.swift
-eval touch $GENERATED_DIR/XCAssets+Generated.swift
 
 swiftgen config run --config Wake\ on\ LAN/Modules/**/swiftgen.yml
 
 SOURCERY_MODULES=(
 	"AddHost"
 	"CoreDataService"
 	"SharedProtocolsAndModels"
+    "WOLResources"
```

Comment:
There is either wrong indentation, or github rendering error. Check locally

Reply:
It looks like it s a GitHub rendering error. I just checked my file locally.